cmd/create/cluster/cmd.go

```diff
@@ -3017,6 +3075,54 @@ func run(cmd *cobra.Command, _ []string) {
 	}
 }
 
+func GenerateContracts(cloudAccounts []*accountsv1.CloudAccount, billingAccount string) ([]*accountsv1.Contract, bool) {
```

Comment:
maybe something like `GetBillingAccountContracts` would be more representative, what do you think?

Reply:
Agreed, changing...

Reply:
Done